src/test/java/hudson/plugins/cobertura/CoberturaPublisherTest.java

```diff
@@ -1,49 +0,0 @@
-package hudson.plugins.cobertura;
```

Comment:
Presumably (@mbarrien would know). I'm putting it back in and restoring the tests - update to PR coming